src/applications/claims-status/tests/components/FilesPage.unit.spec.jsx

```diff
@@ -89,7 +89,7 @@ describe('<FilesPage>', () => {
               },
             },
             closeDate: null,
-            documentsNeeded: false,
```

Comment:
Why did we change this to true? should we have a new expect statement here?

Reply:
In general anytime you have a tracked item with a status of 'NEEDED_FROM_YOU' the `documentsNeeded` flag should be true

Reply:
We don't really use it for any logic in the FilesPage component but I wanted to make sure it was updated since I was updating it in ClaimsStatusPage already anyways